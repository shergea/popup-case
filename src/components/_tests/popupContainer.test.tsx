import React from "react";
import renderer from "react-test-renderer";
import PopupContainer from "../popupContainer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <PopupContainer>
        <div>Popup children</div>
      </PopupContainer>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
