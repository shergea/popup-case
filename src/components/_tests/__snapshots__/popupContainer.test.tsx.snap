// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex h-screen bg-gray-600"
>
  <div
    className="flex flex-row grow"
  >
    <div
      className="m-auto basis-2/4"
    >
      <div>
        Popup children
      </div>
    </div>
  </div>
</div>
`;
